internal/trafficcontrol/tc.go

```diff
@@ -189,9 +199,14 @@ func GetSriovVFs(ifName string) ([]string, error) {
 		if err != nil {
 			return nil, fmt.Errorf("failed to get netlink %s: %q", vfIfName, err)
 		}
-		vfNames = append(vfNames, vfLink.Attrs().Name)
+		vfInfo := &VFInfo{Name: vfLink.Attrs().Name, Index: vfLink.Attrs().Index,
+			PfName: ifName, Authenticated: false}
+		if len(vfLink.Attrs().Vfs) > 0 {
+			vfInfo.VlanID = vfLink.Attrs().Vfs[0].Vlan
```

Comment:
We may need to consider how we recover from the case where we set the VLAN to the port to 4095, then we crash, and we recover - How do we know what the "good" VLAN is in that case? May need a separate story...